tests/kernel/threads/thread_apis/src/test_threads_suspend_resume.c

```diff
@@ -27,6 +27,9 @@ static void threads_suspend_resume(int prio)
 	k_tid_t tid = k_thread_create(&tdata, tstack, STACK_SIZE,
 				      thread_entry, NULL, NULL, NULL,
 				      create_prio, K_USER, K_NO_WAIT);
+	/* checkpoint: resume an unsuspended thread has no effect */
+	k_thread_resume(tid);
+	zassert_false(last_prio == create_prio, NULL);
```

Comment:
how is this check related to k_thread_resume()?

Reply:
Thank you for you comment, you're right. This check has no effect here. I'll change my code.